packages/terra-demographics-banner/src/_utils.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
```

Comment:
When I think of utilities, I think of methods that manipulate arrays and strings as opposed to creating UI widgets. I almost feel like this should be called something like shared objects maybe?